dali/pipeline/data/views.h

```diff
@@ -170,6 +171,18 @@ view_as_tensor(const TensorList<Backend> &data) {
   return { data.template data<U>(), tensor_shape<ndim>(data) };
 }
 
+template <int ndim>
+void to_dims_vec(std::vector<Dims> &dims_vec, const kernels::TensorListShape<ndim> &tls) {
```

Comment:
why not returning the vector ?
And if an output variable, I think that it'd be more intuitive to have the outputs at the end of the arguments

Reply:
I don't want to have a dynamic allocation in steady code path.